pootle/apps/pootle_language/models.py

```diff
@@ -122,6 +122,29 @@ def direction(self):
 
     # # # # # # # # # # # # # #  Methods # # # # # # # # # # # # # # # # # # #
 
+    @classmethod
+    def get_canonical(cls, language_code):
+        """Returns the canonical `Language` object matching `language_code`.
+
+        The matching will be done using a regular expression. If no language can
+        be matched, `None` will be returned.
+
+        :param language_code: the code of the language to retrieve.
+        """
+        try:
+            return cls.objects.get(code__iexact=language_code)
+        except cls.DoesNotExist:
+            _lang_code = language_code
+            if "-" in language_code:
+                _lang_code = language_code.replace("-", "_")
+            elif "-" in language_code:
```

Comment:
This is exactly the same condition as two lines above, and the language code replacement is also identical. I assume this was meant to be a check for `_` and a `.replace('_', '-')`.